<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1774</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="help_info_button">
        <property name="text">
         <string>‚ùìÂú∫ÊôØËØ¥Êòé</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="demo_button">
        <property name="text">
         <string>üì∞ÊºîÁ§∫</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>ÊñáÊ°£ÊâπÂ§ÑÁêÜÂú∫ÊôØ</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="release_info_button">
        <property name="text">
         <string>üí°Âú∫ÊôØÊõ¥Êñ∞‰ø°ÊÅØ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reset_button">
        <property name="text">
         <string>ÈáçÁΩÆÊâÄÊúâ</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="add_docs_button">
              <property name="text">
               <string>‚ûï</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Á¨¨‰∏ÄÊ≠• ÔºöÊãñÊãΩÊàñÁÇπÂáªÊâπÈáè‰∏ä‰º†ÊñáÊ°£</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="step1_help_info_button">
              <property name="text">
               <string>‚ùìÁ¨¨‰∏ÄÊ≠•ÁöÑÂ∏ÆÂä©‰ø°ÊÅØ</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QTableWidget" name="docs_table">
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="autoScroll">
               <bool>true</bool>
              </property>
              <property name="horizontalScrollMode">
               <enum>QAbstractItemView::ScrollPerItem</enum>
              </property>
              <property name="columnCount">
               <number>5</number>
              </property>
              <attribute name="verticalHeaderVisible">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>ÊñáÊ°£ÂêçÁß∞</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Êñá‰ª∂Â§ßÂ∞è</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>‰øÆÊîπÊó∂Èó¥</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Êìç‰ΩúÊåâÈíÆ</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>__ÊñáÊ°£Ë∑ØÂæÑ</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="add_action_button">
              <property name="text">
               <string>‚ûï</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Á¨¨‰∫åÊ≠• ÔºöÊ∑ªÂä†Âä®‰ΩúÊµÅ</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="action_tools_button">
              <property name="text">
               <string>üîß</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="step2_help_info_button">
              <property name="text">
               <string>‚ùìÁ¨¨‰∫åÊ≠•ÁöÑÂ∏ÆÂä©‰ø°ÊÅØ</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="actions_table">
            <property name="contextMenuPolicy">
             <enum>Qt::NoContextMenu</enum>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="autoScroll">
             <bool>true</bool>
            </property>
            <property name="horizontalScrollMode">
             <enum>QAbstractItemView::ScrollPerItem</enum>
            </property>
            <property name="columnCount">
             <number>5</number>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Á±ªÂûã</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Âä®‰Ωú</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Âä®‰ΩúÂÜÖÂÆπ</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Êìç‰ΩúÊåâÈíÆ</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>__Âä®‰Ωúid</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="4,6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QPushButton" name="run_button">
                <property name="text">
                 <string>‚ñ∂ÊâπÈáèÊâßË°å</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Á¨¨‰∏âÊ≠• ÔºöÊâßË°å‰∏é‰∏ãËΩΩ</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="run_log_button">
                <property name="text">
                 <string>üìÑÊâßË°åÊó•Âøó</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="step3_help_info_button">
                <property name="text">
                 <string>‚ùìÁ¨¨‰∏âÊ≠•ÁöÑÂ∏ÆÂä©‰ø°ÊÅØ</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>ÁªìÊûúÊñá‰ª∂È¢ÑËßà</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="result_detail_text">
                <property name="text">
                 <string>üö´ÊâßË°åÊÄªËÆ°ËÄóÊó∂Ôºö--Áßí</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="download_result_button">
                <property name="text">
                 <string>üìÑ‰∏ãËΩΩÁªìÊûú</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTreeView" name="result_tree"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QPushButton" name="debug_button">
                <property name="text">
                 <string>üõ†Ô∏èÂçïÊ≠•Ë∞ÉËØï</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Á¨¨ÂõõÊ≠•ÔºöË∞ÉËØï„ÄêÂèØÈÄâ„Äë</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_9">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="debug_next_button">
                <property name="text">
                 <string>‚û°Ô∏è‰∏ã‰∏ÄÊ≠•</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="step4_help_info_button">
                <property name="text">
                 <string>‚ùìÁ¨¨ÂõõÊ≠•ÁöÑÂ∏ÆÂä©‰ø°ÊÅØ</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>üìÑË∞ÉËØïËÆ∞ÂΩï</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTableWidget" name="actions_with_log_table">
              <property name="contextMenuPolicy">
               <enum>Qt::NoContextMenu</enum>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="autoScroll">
               <bool>true</bool>
              </property>
              <property name="horizontalScrollMode">
               <enum>QAbstractItemView::ScrollPerItem</enum>
              </property>
              <property name="columnCount">
               <number>4</number>
              </property>
              <attribute name="verticalHeaderVisible">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Á±ªÂûã</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Âä®‰Ωú</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Âä®‰ΩúÂÜÖÂÆπ</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Ë∞ÉËØï‰ø°ÊÅØ</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
